slices/nginx-common.yaml

```diff
@@ -9,6 +9,8 @@ slices:
   # handled differently in Chisel.
   # There's also no need (yet) for any init- and systemd-related files.
   config:
+    essential:
+      - apache2_documentroot
```

Comment:
This is ugly, but it was the best solution we could find due to the fact that both the Apache & NGINX packages claim ownership of the `/var/www/html` directory, and chisel does not allow multiple SDFs to claim ownership of the same directory.
We are open to suggestions for a better solution.